prime-router/src/main/kotlin/azure/SendFunction.kt

```diff
@@ -32,19 +32,24 @@ class SendFunction {
 
             val service = Metadata.findService(header.destination)
 
-            //val mockServer = MockSftpServer( 9022 )
+            if( service == null )
+                error( "Unable to find a service handler for ${header.destination}")
 
-            //context.logger.info( "Writing to ${mockServer.getBaseDirectory().toString()}" )
-            //val session = initSshClient()
-            //val sendKlass = Class.forName("gov.cdc.prime.router.SftpSend").kotlin
+            context.logger.info( "Transport found for ${service.fullName} = ${service.transport.type}")
 
-            val transportMetadata: OrganizationService.Transport = lookupTransportMetadata()
-            val transport = SftpTransport() // TODO:  look up the correct class to call based on the transport metadata
+            var transportSuccessful = when( service.transport.type ){
```

Comment:
Just noticed that you are using a non-standard spacing. I'd like to see us standardized on the Kotlin style guide recommendations.

Reply:
I just saw a linter for kotlin - I'll see if I can add that in...